### PR TITLE
feat(PLAT-1810): add an about page using the finalized content

### PR DESCRIPTION
**Pull request by bcamarneiro**
## Why?

Add about page

## How?

- Added About page

## Tickets?

- [PLAT-1810](https://linear.app/fleekxyz/issue/PLAT-1810/add-an-about-page-using-the-finalized-content)

## Contribution checklist?

- [x] The commit messages are detailed
- [x] The `build` command runs locally
- [x] Assets or static content are linked and stored in the project
- [ ] Document filename is named after the slug
- [ ] You've reviewed spelling using a grammar checker
- [ ] For documentation, guides or references, you've tested the commands and steps
- [ ] You've done enough research before writing

## Security checklist?

- [ ] Sensitive data has been identified and is being protected properly
- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The Components are escaping output (to prevent XSS)

## References?

Optionally, provide references such as links

## Preview?
- [DEMO URL](https://skinny-egypt-petite.on-fleek.app/about)
- [PageSpeed Analysis](https://pagespeed.web.dev/analysis/https-skinny-egypt-petite-on-fleek-app-about/5gl1xj3hqp?form_factor=mobile)

image

### CODE REVIEW

File: src/pages/about.astro [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (heldrida) [inline src/pages/about.astro:1]: @bcamarneiro  Looks good! Is it rendering fine without js enabled, correct?

Comment (bcamarneiro) [reply src/pages/about.astro:1]: double checking, thanks! :)

File: Services/Github/api.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+interface Repo {
+  name: string;
+  html_url: string;
+  description: string;
+  contributors_url: string;
+  contributors?: Contributor[];
+}
+
+const DEFAULT_DATA: Repo[] = [
```
Comment (heldrida) [inline Services/Github/api.ts:16]: Let's stick with the default github icon/circle as a fallback. Small, easier to maintain. Good thinking though!

File: src/pages/about.astro (hunk 1 above)
Comment (bcamarneiro) [reply src/pages/about.astro:1]: And correct, checked and it looks good.

File: Services/Github/api.ts (hunk 2 above)
Comment (bcamarneiro) [reply Services/Github/api.ts:16]: done :)

**Review by heldrida — APPROVED** [review]

**Comment by heldrida** [comment]
@bcamarneiro run the code formatter

**Review by heldrida — APPROVED** [review]

**Pull request merged**